go/app/main.go

```diff
@@ -28,14 +45,89 @@ func root(c echo.Context) error {
 func addItem(c echo.Context) error {
 	// Get form data
 	name := c.FormValue("name")
-	c.Logger().Infof("Receive item: %s", name)
+	category := c.FormValue("category")
+	image, err := c.FormFile("image")
+	if err != nil {
+		return err
+	}
+
+	src, err := image.Open()
+	if err != nil {
+		return err
+	}
+	defer src.Close()
+
+	// Create a new SHA256 hash
+	hash := sha256.New()
+
+	// Copy the file content to the hash
+	if _, err := io.Copy(hash, src); err != nil {
+		return err
+	}
+
+	hashInBytes := hash.Sum(nil)
+
+	// Convert hash bytes to hex string
+	hashString := hex.EncodeToString(hashInBytes)
+
+	var image_jpg = hashString + ".jpg"
```

Comment:
[nits] Goで関数内に新しい変数を定義する際は `:=` を使うことが多いです。この場合は:=を使うほうが一般的です
```suggestion
	image_jpg := hashString + ".jpg"
```